src/test/resources/nl/tudelft/ti2206/group9/level/save/saveGameParserTest.json

```diff
@@ -1,2 +1,3 @@
-eyJzZXR0aW5ncyI6eyJzb3VuZEVuYWJsZWQiOnRydWV9LCJjb2lucyI6NDIsImhpZ2hzY29yZSI6
-eyJzY29yZSI6MjU2MH0sInBsYXllcm5hbWUiOiJNYXRoaWFzIn0=
\ No newline at end of file
+eyJzb3VuZHRyYWNrc2V0dGluZ3MiOnsic291bmR0cmFja0VuYWJsZWQiOnRydWV9LCJjb2lucyI6
+NDIsImhpZ2hzY29yZSI6eyJzY29yZSI6MjU2MH0sInNvdW5kRWZmZWN0c3NldHRpbmdzIjp7InNv
+dW5kRWZmZWN0c0VuYWJsZWQiOnRydWV9LCJwbGF5ZXJuYW1lIjoiTWF0aGlhcyJ9
```

Comment:
I think these are `.ses` files, not `.json` files. I know @MathiasMeuleman should've changed this, but apparently this got slipped through. Could you still change it please?

<review, naming